Assets/LeapMotion/Scripts/Utils/LeapUnityExtensions.cs

```diff
@@ -98,7 +98,7 @@ public static class UnityMatrixExtension {
      * @param mirror If true, the operation is reflected along the z axis.
      */
     public static Vector3 Translation(this Matrix matrix, bool mirror = false) {
-      return matrix.TransformPoint(LEAP_ORIGIN).ToUnityScaled();
+      return matrix.TransformPoint(LEAP_ORIGIN).ToVector3();
```

Comment:
This substantially changes the way the function works -- it no longer converts the coordinate system. However, there isn't a clear winner between the old way and the new way -- if the caller has a matrix from an already transformed frame, they want the new version, if they have one from a pre-transformed frame, they want the old way.

Also, it would be a bit more efficient to implement this as:

    return matrix.origin.ToVector3();

or

    return matrix.origin.ToUnityScaled();